README.md

```diff
@@ -8,9 +8,11 @@ This terraform deploys an RDS instance.
 module "rds" {
   source = "github.com/byu-oit/terraform-aws-rds?ref=v0.2.0"
```

Comment:
change the docs and example to use the next version. so maybe in this case a `v0.2.1` or something like that